packages/plugins/users-permissions/admin/src/pages/Providers/index.jsx

```diff
@@ -63,7 +63,7 @@ export const ProvidersPage = () => {
 
   const submitMutation = useMutation((body) => put('/users-permissions/providers', body), {
     async onSuccess() {
-      await queryClient.invalidateQueries(['users-permissions', 'providers']);
+      await queryClient.invalidateQueries(['users-permissions', 'get-providers']);
```

Comment:
Was that a typo? Don't we have a type for the invalid queries to help up with that?

Reply:
Yes, this is old queries that didn't get migrated and if you look at the query above in the code the cache tag is `get-providers` :)